app/src/main/java/org/greenstand/android/TreeTracker/activities/ImageCaptureActivity.kt

```diff
@@ -16,20 +16,17 @@ import androidx.camera.core.Preview
 import androidx.camera.core.PreviewConfig
 import java.io.File
 import org.greenstand.android.TreeTracker.R
-import org.greenstand.android.TreeTracker.usecases.CaptureTreeLocationUseCase
 import org.greenstand.android.TreeTracker.utilities.AutoFitPreviewBuilder
 import org.greenstand.android.TreeTracker.utilities.ImageUtils
 import org.greenstand.android.TreeTracker.utilities.ValueHelper
 import org.greenstand.android.TreeTracker.viewmodels.NewTreeViewModel.Companion.FOCUS_THRESHOLD
-import org.koin.android.ext.android.inject
 import timber.log.Timber
 
 class ImageCaptureActivity : AppCompatActivity() {
 
     private lateinit var viewFinder: TextureView
     private lateinit var imageCaptureButton: ImageButton
     private lateinit var toolbarTitle: TextView
-    private val captureTreeLocationUseCase: CaptureTreeLocationUseCase by inject()
```

Comment:
These objects are not necessary for now. But they may be introduced in subsequent PRs when adding tree-uuids. This change is specific to this PR given the requirement.